### PR TITLE
[5] Display market price and est. exec price on Limit orders table

### PR DESCRIPTION
**Pull request by shoom3301**
# Summary

image

image

### Fixes
 - Don't close tooltip when cursor is on content
 - Don’t update orders when the window is not active
 - Set the Smart quote selection only once in RateInfo
 - Execution time for history orders
 - Inverted fee amount
 - Fix ExecutionPriceTooltip layout
 - Fix OrderContextMenu styles

### CODE REVIEW

File: src/cow-react/modules/limitOrders/utils/calculateExecutionPrice.ts [hunk 1]
```diff
@@ -50,13 +50,13 @@ export function calculateExecutionPrice(params: ExecutionPriceParams): Price<Cur
   const marketPrice = new Price({
     baseAmount: inputCurrencyAmount,
     quoteAmount: convertAmountToCurrency(
-      inputCurrencyAmount.multiply(marketRate).subtract(feeAmount),
+      inputCurrencyAmount.multiply(marketRate).add(feeAmount),
```
Comment (shoom3301) [inline src/cow-react/modules/limitOrders/utils/calculateExecutionPrice.ts:53]: Looks like it was wrong before

**Comment by anxolin** [comment]
looks nice! i still need to go over a deeper review. One small feedback. 

We currently allow to invert the price by clicking on it. 
Now we have two columns with the price, i think we should make the inversion of the price affect the whole row, and not just the limit price cell

otherwise you end up in this situation

image

**Comment by elena-zh** [comment]
> looks nice! i still need to go over a deeper review. One small feedback.
> 
> We currently allow to invert the price by clicking on it. Now we have two columns with the price, i think we should make the inversion of the price affect the whole row, and not just the limit price cell
> 
> otherwise you end up in this situation
> 
> image

[There ](https://www.figma.com/file/9UJGrVP7AYtSa3M35uQPK5?node-id=2027:5315#363340124) was discussion about invert prices issue. 

> One idea here, is to not have the 'revert' icon. Instead this is controlled in the 'Limit price' column, just like the user is used to in the Limit order (form) container.

**Comment by elena-zh** [comment]
Hey @shoom3301 , nice!
And now, issues that seem weird to me :)
1. Market Buy order: Est. execution price shows a weird number (seems like it is a To token: in my case it is GNO, with the COW Symbol

2. Limit Buy order: Est. execution price is in opposite token in comparison with Limit price column; 
shows weird prices that do not correspond to the price on the Limit order form
image
3. Limit Sell order: Est. execution price is different in comparison with the Limit order form. Is this OK?
image 
diff estimated execution prices

4. Est. execution price is loaded up to 15 seconds after an order placement. Is this OK?

5. Not sure if this is related, but I noticed different rounding rules on the form and inside the table
diff rounding

Could you please take a look into these issues?
Thanks!

**Pull request merged**